vhdbuilder/packer/components.json

```diff
@@ -304,6 +304,7 @@
       "downloadURL": "mcr.microsoft.com/oss/open-policy-agent/gatekeeper:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
+        "v3.14.0",
         "v3.13.4",
         "v3.13.3"
```

Comment:
do we need to keep the oldest version of these images cached even after this addition?

Reply:
After the update to the new image version has rolled out (hoping to make the Feb AKS release) I'll circle back with a PR removing the old one.

Reply:
Went ahead and removed 3.13.4 and 1.2.2 since those didn't get deployed due to the code freeze, hope that helps